modules/sno-du-conn-disabling-ntp.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-connected.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-conn-disabling-ntp_{context}"]
+= Disabling Network Time Protocol (NTP)
+
+After the system is configured for Precision Time Protocol (PTP), you need to disable NTP to prevent it from impacting the system clock.
+
+.Procedure
+
+* No configuration changes are needed. Use the provided settings:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: disable-chronyd
```

Comment:
I no longer see this in source-cr. 

I think disabling chronyd is removed via this PR: https://github.com/openshift-kni/cnf-features-deploy/commit/1df5df81beb4d73fd0811a930ddec9c7cd22af79
 
I think this can be done as a separate doc PR if preferred since this change id recently added via 4.10.z release.